misc/misc.go

```diff
@@ -0,0 +1,26 @@
+package misc
+
+import (
+	"crypto/rand"
+	"math/big"
+)
+
+const seed = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-"
```

Comment:
Usually alphanumeric includes `_` instead of `-`, if it includes any. Can you document the rationale for using `-` and not using `_`?

Reply:
No rationale in particular, I was bothered by `-` when I copied this seed from the internet. With your "vote" in I will make it a `_`.
Thanks for the catch!

Reply:
alternatively you could just not have `-` or `_` unless there is a reason for it.

Reply:
True that. Additionally, at the moment this function is used to create IDs for metrics, which is relatively a non-critical task, thus we can afford such libereties.